controllers/handlers/wasp-agent/rbac.go

```diff
@@ -0,0 +1,113 @@
+package wasp_agent
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	hcov1beta1 "github.com/kubevirt/hyperconverged-cluster-operator/api/v1beta1"
+	"github.com/kubevirt/hyperconverged-cluster-operator/controllers/operands"
+)
+
+func NewClusterRoleBindingHandler(Client client.Client, Scheme *runtime.Scheme) operands.Operand {
+	return operands.NewConditionalHandler(
+		operands.NewClusterRoleBindingHandler(Client, Scheme, newClusterRoleBinding),
+		shouldDeployWaspAgent,
+		func(hc *hcov1beta1.HyperConverged) client.Object {
+			return newClusterRoleBindingWithNameOnly(hc)
+		},
+	)
+}
+
+func newClusterRoleBinding(hc *hcov1beta1.HyperConverged) *rbacv1.ClusterRoleBinding {
+	return &rbacv1.ClusterRoleBinding{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: rbacv1.SchemeGroupVersion.String(),
+			Kind:       "ClusterRoleBinding",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   clusterRoleName,
+			Labels: operands.GetLabels(hc, AppComponentWaspAgent),
+		},
+		RoleRef: rbacv1.RoleRef{
+			Kind:     "ClusterRole",
+			Name:     clusterRoleName,
+			APIGroup: "rbac.authorization.k8s.io",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      "ServiceAccount",
+				Name:      waspAgentServiceAccountName,
+				Namespace: hc.Namespace,
+			},
+		},
+	}
+}
```

Comment:
we can reuse `newClusterRoleBindingWithNameOnly` here; for example:
```suggestion
	crb := newClusterRoleBindingWithNameOnly(hc)
	
	crb.RoleRef = rbacv1.RoleRef{
		Kind:     "ClusterRole",
		Name:     clusterRoleName,
		APIGroup: "rbac.authorization.k8s.io",
	}
	
	crb.Subjects = []rbacv1.Subject{
		{
			Kind:      "ServiceAccount",
			Name:      waspAgentServiceAccountName,
			Namespace: hc.Namespace,
		},
	}
	
	return crb
}
```
WDYT?

Reply:
@nunnatsa Sure, fixed.